WORKSPACE

```diff
@@ -31,6 +31,23 @@ http_archive(
 load("@io_bazel_rules_sass//sass:sass_repositories.bzl", "sass_repositories")
 sass_repositories()
 
+# Add Angular source and Bazel rules.
+http_archive(
+  name = "angular",
+  # Temporarily locked down to the angular/angular:bazel branch. This branch includes necessary
+  # commits that make building from source possible.
```

Comment:
Is there an issue somewhere tracking this?

Reply:
Not that I know of. I think it's just a WIP branch that includes the new design with `rules_angular_dependencies`.